frontend/src/components/orders/OverdueSummary.vue

```diff
@@ -0,0 +1,78 @@
+<script setup lang="ts">
+import Dropdown from "primevue/dropdown";
+import { ref, watch } from "vue";
+
+import { updateOverdueNotice } from "@/backendClient";
+import OrderDocuments from "@/components/orders/OrderDocuments.vue";
+import SubOrderItemButton from "@/components/orders/SubOrderItemButton.vue";
+import useConfirmations from "@/compositions/useConfirmations";
+import useNotifications from "@/compositions/useNotifications";
+import { formatIsoDateString } from "@/global/helpers";
+import {
+  DocumentKind,
+  OverdueNoticeLevel,
+  OverdueNoticePaymentStatus,
+} from "@/global/types/appTypes";
+import type { OverdueNotice } from "@/global/types/entities";
+
+const props = defineProps<{
+  overdue: OverdueNotice;
+}>();
+const notifications = useNotifications();
+
+const confirm = useConfirmations();
+const overdueType = Object.values(OverdueNoticeLevel);
+const paymentStatus = Object.values(OverdueNoticePaymentStatus);
+let noticeLevel = ref(props.overdue.notice_level);
+let overduePaymentStatus = ref(props.overdue.payment_status);
+
+watch(noticeLevel, async () => {
+  confirm.showUpdateOverdueNoticeLevelConfirmation(async () => {
+    await updateOverdueNotice(props.overdue.id, {
+      notice_level: noticeLevel.value,
+    });
+    notifications.showUpdateOverdueNoticeLevelNotification();
+  });
+});
+</script>
+
+<template>
+  <section
+    class="flex flex-col justify-items-start gap-2 sm:flex-row sm:gap-8 sm:items-center flex-wrap"
+  >
+    <p>
+      <span class="font-bold">Zahlungen berücksichtigt bis: </span>
+      {{ formatIsoDateString(overdue.payments_until) }}
+    </p>
+    <p>
+      <span class="font-bold">Mahndatum: </span>
+      {{ formatIsoDateString(overdue.notice_date) }}
+    </p>
+    <p><span class="font-bold">Mahnstufe:</span></p>
+    <Dropdown
+      id="overdueType"
+      v-model="noticeLevel"
+      :options="overdueType"
+      placeholder="Anrede"
```

Comment:
the placeholder seems wrong. But do we even need a placeholder if we have a label?